codec-http2/src/test/java/io/netty/handler/codec/http2/HpackDecoderTest.java

```diff
@@ -160,27 +164,13 @@ public void testDecodeULE128LongOverflow2() throws Http2Exception {
         }
     }
 
-    @Test
-    public void testSetTableSizeWithMaxUnsigned32BitValueSucceeds() throws Http2Exception {
-        byte[] input = {(byte) 0x3F, (byte) 0xFF, (byte) 0xFF, (byte) 0xFF, (byte) 0xFF, (byte) 0x0E};
-        ByteBuf in = Unpooled.wrappedBuffer(input);
-        try {
-            final long expectedHeaderSize = 4026531870L; // based on the input above
-            hpackDecoder.setMaxHeaderTableSize(expectedHeaderSize);
-            hpackDecoder.decode(0, in, mockHeaders);
```

Comment:
is there a reason for the method movement? if not, can you undo this movement of this unit test and just add the extra parameter here?

Reply:
good catch, I'll move that back.

Reply:
- [x] Move